tests/test_core.py

```diff
@@ -2259,6 +2259,7 @@ def test_runtimelink(self):
   def can_dlfcn(self):
     if Settings.ALLOW_MEMORY_GROWTH == 1: return self.skip('no dlfcn with memory growth yet')
     if self.is_wasm_backend(): return self.skip('no shared modules in wasm backend')
+    self.banned_js_engines = [V8_ENGINE, NODE_JS] # V8 doesn't support mismatch between table import decl initial size and imported table runtime size
```

Comment:
Is the issue https://bugs.chromium.org/p/v8/issues/detail?id=5795 ? then maybe a link here, so if we forget later it's easy to find?

why disable node.js? we do need it here, for the non-wasm dlfcn tests, which do run in node. is the issue that until v8 supports this node won't either? if so we could ban node.js conditional on `is_wasm`.